homeassistant/components/vesync/fan.py

```diff
@@ -90,12 +90,7 @@ def _setup_entities(
     entities = []
     for dev in devices:
         if DEV_TYPE_TO_HA.get(SKU_TO_BASE_DEVICE.get(dev.device_type, "")) == "fan":
-            entities.append(VeSyncFanHA(dev, coordinator))
-        else:
-            _LOGGER.warning(
-                "%s - Unknown device type - %s", dev.device_name, dev.device_type
```

Comment:
This warning is what is displayed whenever a humidifier shows up.   Removing since now they will have two sensors,  after the humidifier platform merged well featured.  I also personally have found this warning annoying as it clutters my logs.

Reply:
[VeSyncFanHA(dev, coordinator) for dev in devices if DEV_TYPE_TO_HA.get(SKU_TO_BASE_DEVICE.get(dev.device_type, "")) == "fan"]

Reply:
Adjusted.